lib/Proxy.js

```diff
@@ -173,8 +177,7 @@ define([
 
 		_publishInSequence: function (message) {
 			var session = this._getSession(message.sessionId);
-
-			if (message.sequence <= session.lastSequence) {
+			if (!isFatalErrorFromPreExecutor(message) && message.sequence <= session.lastSequence) {
```

Comment:
An alternative would be to generate valid sequence numbers for fatal error messages, but I didn't quite figure out how to implement this.